ts/Series/TiledWebMap/TiledWebMapSeries.ts

```diff
@@ -438,7 +447,8 @@ addEvent(MapView, 'beforeMapViewInit', function (e: any): boolean {
         const ProviderDefinition =
             TilesProvidersRegistry[twm.provider.type],
             def = new ProviderDefinition(),
-            providerProjectionName: string = def.getProjectionName();
+            providerProjectionName: string = def.getProjectionName(),
+            { minZoom, maxZoom } = def.getMinMaxZoom();
```

Comment:
I'm wondering if this `getMinMaxZoom` function is necessary? It's identical for all the provider definitions, so in its current form it seems like a waste of space. Without it, we could just do `{ minZoom, maxZoom } = def;`.